tests/unit/onyxTest.js

```diff
@@ -24,11 +24,37 @@ Onyx.init({
 describe('Onyx', () => {
     let connectionID;
 
+    /** @type OnyxCache */
+    let cache;
+
+    beforeEach(() => {
+        cache = require('../../lib/OnyxCache').default;
```

Comment:
Why is this required for each test and not imported at the top of the file?

Reply:
The OnyxCache resolves to undefined if we import it directly from the top of the file. I took this reference of using require() from existing unit tests found here and used same test logic.

https://github.com/Expensify/react-native-onyx/blob/7d6cdc339acc17aa90f50f61e3e825364163146b/tests/unit/onyxCacheTest.js#L17
https://github.com/Expensify/react-native-onyx/blob/af02f32b74ae7c870b75e029d3a16fe4079442cd/tests/unit/onyxClearWebStorageTest.js#L32
https://github.com/Expensify/react-native-onyx/blob/5686d24c198204d023a78c0780bb37850335cce6/tests/unit/onyxClearNativeStorageTest.js#L25

Reply:
Ah, that's interesting. If other tests are doing it, I guess that's OK for now. Thanks!